templates/tapestry.js

```diff
@@ -44,7 +44,7 @@ function tapestryTool(config){
         simulation,                                             // Force
         tapestrySlug, 
         saveProgress = true,                                    // Saving Progress
-        tapestryDimensionsBeforeDrag, nodeBeforeDrag,
+        tapestryDimensionsBeforeDrag, nodesBeforeDrag,
```

Comment:
Doesn't seem like this is used at all.

Reply:
Not sure how you got that idea - both variables seem to be used to me.

Reply:
`nodesBeforeDrag` is not used. Can you point out where it's used to me?

Reply:
Sorry I see you are now using it. Before it was not being used (only somewhere commented out).